package.json

```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "test": "npm run unit && npm run lint",
-    "unit": "NODE_ENV=test istanbul cover _mocha",
+    "unit": "LOG_LEVEL=error NODE_ENV=test istanbul cover _mocha",
     "unit:nocov": "NODE_ENV=test mocha",
```

Comment:
We probably want the logs silenced for the `nocov` script too. And I don't think `NODE_ENV` needs to be set now.